torch/fx/_symbolic_trace.py

```diff
@@ -328,7 +328,6 @@ def create_arg(self, a: Any) -> "Argument":
             for n, p in self.root.named_parameters():
                 if a is p:
                     return self.create_node("get_attr", n, (), {})
-            raise NameError("parameter is not a member of this module")
```

Comment:
This seems technicaly legal, but is it a good idea to pass a parameter from another nn.Module?

What's the use-case?

Reply:
I'm writing a onnx2torch converter using torch fx, if one Add node in onnx has two inputs, one is from another layer's output, and one is constant with both shape (N,C,H,W), so the constant has to be passed to the whole graph module(nn.Module), if created as tensor, then it cannot be moved to cuda, so parameter is used.

Reply:
Hmm sounds good. But you have to change the doc comments regarding Parameter.

Reply:
How about just delete those comments.

Reply:
I was referring to

```
* For a Parameter, emit a ``get_attr`` node referring to that Parameter
            * For a non-Parameter Tensor, store the Tensor away in a special
              attribute referring to that attribute.
```

Reply:
How about this one